NuGet.config

```diff
@@ -8,6 +8,6 @@
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="benchmarkdotnet-ci-feed" value="https://ci.appveyor.com/nuget/benchmarkdotnet" />
-    <add key="dotnet-myget-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+    <add key="aspnet-aspnetcore" value="https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore/index.json" />
```

Comment:
Today I have downloaded the latest dotnet core sdk and installed it on my box and I got following error when doing dotnet restore:

```log
PS C:\Projects\performance\src\benchmarks\real-world\Roslyn> dotnet restore
  Restore completed in 16,19 ms for C:\Projects\performance\src\tools\Reporting\Reporting\Reporting.csproj.
  Restore completed in 24,14 ms for C:\Projects\performance\src\harness\BenchmarkDotNet.Extensions\BenchmarkDotNet.Extensions.csproj.
C:\Projects\performance\src\benchmarks\real-world\Roslyn\CompilerBenchmarks.csproj : error NU1102: Unable to find package Microsoft.AspNetCore.App.Ref with version (= 3.0.0-preview7.19327.11)
C:\Projects\performance\src\benchmarks\real-world\Roslyn\CompilerBenchmarks.csproj : error NU1102:   - Found 133 version(s) in https://dotnet.myget.org/F/dotnet-core/api/v3/index.json [ Nearest version: 3.0.0-preview4-19177-01 ]
C:\Projects\performance\src\benchmarks\real-world\Roslyn\CompilerBenchmarks.csproj : error NU1102:   - Found 28 version(s) in dotnet-core [ Nearest version: 3.0.0-preview4-19121-14 ]
C:\Projects\performance\src\benchmarks\real-world\Roslyn\CompilerBenchmarks.csproj : error NU1102:   - Found 4 version(s) in api.nuget.org [ Nearest version: 3.0.0-preview6.19307.2 ]
C:\Projects\performance\src\benchmarks\real-world\Roslyn\CompilerBenchmarks.csproj : error NU1102:   - Found 0 version(s) in benchmarkdotnet-ci-feed
C:\Projects\performance\src\benchmarks\real-world\Roslyn\CompilerBenchmarks.csproj : error NU1102:   - Found 0 version(s) in https://dotnet.myget.org/F/sourcelink/api/v3/index.json
C:\Projects\performance\src\benchmarks\real-world\Roslyn\CompilerBenchmarks.csproj : error NU1102:   - Found 0 version(s) in https://dotnet.myget.org/F/symreader-converter/api/v3/index.json
C:\Projects\performance\src\benchmarks\real-world\Roslyn\CompilerBenchmarks.csproj : error NU1102:   - Found 0 version(s) in https://dotnet.myget.org/F/symreader/api/v3/index.json
C:\Projects\performance\src\benchmarks\real-world\Roslyn\CompilerBenchmarks.csproj : error NU1102:   - Found 0 version(s) in https://www.myget.org/F/xunit/api/v3/index.json
```

It has suprised me, becasue we don't have any dependency to ASP.NET stuff.

According to https://github.com/dotnet/core/issues/2924 we need to add this feed to be able to use daily SDK builds.

@billwert I have also removed the myget feed from the list because as far I know we have moved from `https://dotnet.myget.org/F/dotnet-core/api/v3/index.json` to `https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json`

Reply:
cool. I didn't hear about this - was it announced somewhere? (I'd like to make sure I'm plugged into whatever channel news like that comes through.)

Reply:
> was it announced somewhere?

I found out about it when some of my stuff stopped working ;)

@ViktorHofer is there any news channel for changes related to NuGet feeds?

Reply:
This is far to specific for a news channel. This was announced in the .NET Engineering Partners DG.